node/src/Router.ts

```diff
@@ -1049,14 +1071,22 @@ export class Router extends EnhancedEventEmitter<RouterEvents>
 		if (appData && typeof appData !== 'object')
 			throw new TypeError('if given, appData must be an object');
 
-		const internal = { ...this.#internal, rtpObserverId: uuidv4() };
-		const reqData = { maxEntries, threshold, interval };
+		const reqData = {
+			rtpObserverId : uuidv4(),
+			maxEntries,
+			threshold,
+			interval
+		};
```

Comment:
Not sure why ESLint doesn't catch these things, it should.

```suggestion
		const reqData =
		{
			rtpObserverId : uuidv4(),
			maxEntries,
			threshold,
			interval
		};
```